### ISSUE TITLE
Throws error when using reserved MySQL words/statements

### ISSUE THREAD

**Issue by larskoole** [beyondcode/invoker-community#227]
In this case, we have a column named 'usage' which Invoker can't handle because it's a reserved word.

The query should probably be using backticks ` to prevent this.

------
**OS**: darwin
**Invoker Version**: 2.6.0
**Laravel Version**: 8.52.0
**Local project**: true
**PHP Binary**: /usr/local/bin/php
**Route**: /project/index?class=App%5CModels%5CUser&time=1627653393798&project=local-uHVRRCqER32HNUiTDvvdgJ

**Error**:
```
Illuminate\Database\QueryException 

  SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'usage WHERE Field = 'type'' at line 1 (SQL: SHOW COLUMNS FROM usage WHERE Field = 'type')

  at /vendor/laravel/framework/src/Illuminate/Database/Connection.php:692
    688▕         // If an exception occurs when attempting to run a query, we'll format the error
    689▕         // message to include the bindings with SQL, which will make this exception a
    690▕         // lot more helpful to the developer instead of just the database's errors.
    691▕         catch (Exception $e) {
  ➜ 692▕             throw new QueryException(
    693▕                 $query, $this->prepareBindings($bindings), $e
    694▕             );
    695▕         }
    696▕     }

      [2m+8 vendor frames [22m
  9   phar:/Applications/Invoker.app/Contents/Resources/invoker.phar/src/Factories/AttributeItem.php:37
      Illuminate\Database\Connection::select("SHOW COLUMNS FROM usage WHERE Field = 'type'")

  10  phar:/Applications/Invoker.app/Contents/Resources/invoker.phar/src/Factories/AttributeItem.php:25
      _HumbugBox66505c3d0109\Invoker\Factories\AttributeItem::getValues(Object(App\Models\Usage), Object(Doctrine\DBAL\Schema\Column))
```

**Comment by larskoole** [comment]
I've gotten a .phar from @mpociot which fixes the issue. I'll keep the issue open till someone from your team closes it :)

**Comment by mpociot** [comment]
This is now fixed in version 2.6.1
